src/runtime_src/core/common/api/module_int.h

```diff
@@ -33,7 +40,8 @@ patch(const xrt::module&, const std::string& argnm, size_t index, const xrt::bo&
 // of the control code buffer.
 XRT_CORE_COMMON_EXPORT
 void
-patch(const xrt::module&, uint8_t*, size_t*, const std::vector<std::pair<std::string, uint64_t>>*);
+patch(const xrt::module&, uint8_t*, size_t*, const std::vector<std::pair<std::string, uint64_t>>*,
```

Comment:
It would be good to document the arguments, in particular the new `index` argument.